h/models/group.py

```diff
@@ -66,6 +91,14 @@ class Group(Base, mixins.Timestamps):
     writeable_by = sa.Column(sa.Enum(WriteableBy, name='group_writeable_by'),
                              nullable=True)
 
+    @hybrid_property
```

Comment:
What were you expecting `@hybrid_property` to do here?

As I understand it, the [purpose of `@hybrid_property`](https://docs.sqlalchemy.org/en/latest/orm/extensions/hybrid.html) is to produce properties which can be evaluated either against an instance, to get the value of the property for that instance, or against the class, in which case they can be used to generate SQL expressions, eg. for loading instances where the property has a certain value from the database.

The first case works fine here, but unlike say `models.User.userid`, the latter case doesn't work properly - that is you can't do something like `session.query(models.Group).filter(models.Group.groupid == "groupid:foo@bar.com")`. In the context of `userid` this is handled using the custom comparator.

If you don't care about the inability to generate SQL expressions then you can just use the Python `@property` decorator. See `models.Annotation.is_reply` for example.

Reply:
:+1: